src/redux/features/deliverables/deliverablesSelectors.ts

```diff
@@ -1,6 +1,8 @@
 import { RootState } from 'src/redux/rootReducer';
 
-export const selectDeliverableList = (organizationId: number) => (state: RootState) =>
-  state.deliverableList[organizationId];
+export const selectDeliverablesSearchRequest = (requestId: string) => (state: RootState) =>
+  state.deliverablesSearch[requestId];
 
 export const selectDeliverable = (deliverableId: number) => (state: RootState) => state.deliverables[deliverableId];
+
+export const selectDeliverableFetchRequest = (requestId: string) => (state: RootState) => state.deliverables[requestId];
```

Comment:
I kept the ability to select the request, to check for status... but I don't really know if it is needed. What do you think? I had both in there originally because it seemed like a waste using the async thunk creator without it but it seems unnecessary, like you're pointing out

Reply:
I did see that.
My thinking is the client will select the result and handle the status vs data accordingly. I have been doing something similar in existing code.
One thought, we could keep the one that returns the complex payload and add the other selector (for just the data) as we find  a need?

Reply:
Cool, I left them both in there.